skeleton/webpack/aurelia_project/tasks/run.ext

```diff
@@ -1,8 +1,39 @@
 import { NPM } from 'aurelia-cli';
+import { CLIOptions } from 'aurelia-cli';
+import * as project from '../aurelia.json';
 
-export default function() {
+var kill = require('tree-kill');
+
+const port = CLIOptions.getFlagValue('port') || project.platform.port;
+const host = CLIOptions.getFlagValue('host') || project.platform.host || "localhost";
+
+const npm =  new NPM();
+
+const run = () => {
   console.log('`au run` is an alias of the `npm start`, you may use either of those; see README for more details.');
 
   const args = process.argv.slice(3);
-  return (new NPM()).run('start',['--', ...args]);
+
+  if (!CLIOptions.hasFlag('port')) {
+    args.push('--port');
+    args.push(port);
+  }
+  if (!CLIOptions.hasFlag('host')) {
```

Comment:
This two extra `--port` and `--host` destroyed normal `au run --open`.

Failed at duplicated args in
`webpack-dev-server --env.extractCss  --port 5000 "--open" "--port" "5000" "--host" "localhost"`

We should modify webpack.config.js to get port/host from aurelia.json
```
devServer: {
    ...
    port: project.platform.port || 5000,
    host: project.platform.host || 'localhost'
  },
```

Then we don't need the code here.

Reply:
It looks like we also need to remove `--port 5000` from package.json.
```
"start": "webpack-dev-server --env.extractCss  --port 5000",
```

Reply:
> This two extra `--port` and `--host` destroyed normal `au run --open`.
> 
> Failed at duplicated args in
> `webpack-dev-server --env.extractCss --port 5000 "--open" "--port" "5000" "--host" "localhost"`
> 
> We should modify webpack.config.js to get port/host from aurelia.json
> 
> ```
> devServer: {
>     ...
>     port: project.platform.port || 5000,
>     host: project.platform.host || 'localhost'
>   },
> ```
> 
> Then we don't need the code here.

`--open` works fine for me, and when I removed the code here and add it to the `devServer` section then overriding the port and host does not work properly, the webpack said it is running on the mentioned port and host but they did not, since `--open` now works and `Failed at duplicated args in` this has been handled, I guess it is good to leave the code here.

I think you were testing with `.net core` setting, since port `5000` is only used there, right? 
If so, I have tested with that configuration again, It ran without error for me for the following commands:
```
au cypress --port 8090 --host 127.0.0.1 --open --start --run 
```
```
au run --open --port 8090 --host 127.0.0.1 
```
```
dotnet run 
```

Reply:
Yes, I was testing donet template.

Reply:
I've tested that with `cypress`, looks to be working fine :smile:

Reply:
You have not updated `webpack.config.js` to read port and host from aurelia.json.

Reply:
Yes, I said above that it was not working, I'll give it a time tonight again. 

"the webpack said it is running on the mentioned port and host but they did not" <- this happened when I passed `--port` and `--host` as arguments, but normally it works on port `8080`